src/guest/start.c

```diff
@@ -513,13 +534,40 @@ static void cleanup_passthrough(void) {
 	}
 }
 
+static int check_soundcard_on_host(){
+        int ret = system("cat /proc/asound/cards | grep sof");
+        return ret;
+}
+
+static int insert_sof_tgl_snd_module() {
+	int pid;
+	int wst;
+	pid = fork();
```

Comment:
use this function to load kernel mods: load_kernel_module(char *mod_name), you can find it in utils.c